### ISSUE TITLE
InFocusDisplay Always Showing Alert

### ISSUE THREAD

**Issue by rgomezp** [OneSignal/OneSignal-iOS-SDK#600]
**Description:**
Multiple reports in wrapper SDKs that InFocusDisplay is not functioning correctly. It seems this was introduced in 2.12.2. 

**Environment**

1. Cordova - 2.8.0 (native iOS 2.12.2)
2. React Native - 3.6.0 (native iOS 2.12.2)

**Steps to Reproduce Issue:**
See issues: 
   - https://github.com/OneSignal/OneSignal-Cordova-SDK/issues/597
   - https://github.com/OneSignal/react-native-onesignal/issues/918

**Comment by ludufre** [comment]
With release 2.8.2 the issue continues...

### Stack

```
   CocoaPods : 1.8.4
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
    RubyGems : 3.0.3
        Host : Mac OS X 10.15.2 (19C57)
       Xcode : 11.3.1 (11C504)
         Git : git version 2.21.1 (Apple Git-122.3)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : cocoapods - git - https://github.com/CocoaPods/Specs.git @ ab35ade6ae00986f4bdcde525a5ce68e97468cf2
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# DO NOT MODIFY -- auto-generated by Apache Cordova
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '11.0'

target 'Sonar' do
	project 'Sonar.xcodeproj'
	pod 'GoogleMaps', '~> 3.1.0'
	pod 'OneSignal', '2.12.5'
end
```

**Comment by gordon-ghk** [comment]
I am using ionic, same problem here
pod 'OneSignal', '2.12.2'

**Comment by rgomezp** [comment]
This issue has been resolved in 2.12.6
